styles.css

```diff
@@ -10,6 +10,9 @@ body{
 header, footer, aside, .main{
   padding: 1em;
 }
+header, footer, aside, .main > *{
```

Comment:
You have the right idea but you would need to add `> *` to each of the tags, so something like...

```css
header > *, footer > *, aside > *, .main > * {
  /* style goes here */
}
```